### PR TITLE
Feat/css

### PR DESCRIPTION
**Pull request by AsherSpurr**
# Pull Request

## Description
 This PR will add styling to the nav bar and some of the Home component

## Screenshots (if applicable)
**Intro/Welcome section** 
Screenshot 2024-04-04 at 11 58 20 AM

**Tea Types**
Screenshot 2024-04-04 at 11 58 37 AM

## Changes Made
**Added:**
- styling to Nav bar
- some styling to the Home component
- a 'welcome' section at the top, where we can explain what Spilled is, the goal of the site, and how to use it.

**Removed:**
- default styling in App.css

**Note about other files changed**
- **This PR was made at a local branch that kept the code we were all working on that night. So unfortunately the non-working code is included in this. I can delete it from this PR if needed.**

## Checklist- 

- [ ] The code follows the project's coding standards.
- [ ] Unit tests have been added or updated to cover the changes.
- [x] Documentation has been updated to reflect the changes (if applicable).
- [ ] The code compiles without errors.
- [ ] The changes have been tested locally and pass all relevant tests.
- [ ] All new and existing tests pass.
- [x] The pull request has been reviewed by at least one other contributor.

## Reviewer Instructions
Leave a comment requesting a change if we would like the non-working code deleted from this branch.

### CODE REVIEW

**Review by corysanders3 — APPROVED** [review]
Really nice updates to the home page. Really liking the overall feel of the site so far.

File: src/components/home/Home.tsx [hunk 1]
```diff
@@ -1,12 +1,89 @@
...
+          <div className="info">
+            <h2 className="home-h2">Home</h2>
+            <p>
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+              eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
+              ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+              aliquip ex ea commodo consequat.
+            </p>
+            <button className="infoButton">
+              <NavLink to="/tea/black">See Black tea's →</NavLink>
```
Comment (tednaphil) [inline src/components/home/Home.tsx]: Update the button text for each tea be updated to omit the apostrophe

File: src/components/nav/Nav.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+}
+
+a {
+    text-decoration: none;
+    font-size: 1em;
+    color: #010101;
+    padding: 5px;
+}
+
+a:hover {
```
Comment (tednaphil) [inline src/components/nav/Nav.css:20]: LOVE hover styling! For our next iteration, how would yall feel about adding rules for `.active` so the nav bar reflects which page the user is on?

**Review by tednaphil — CHANGES_REQUESTED** [review]
Only change needed is for the home page tea buttons that are possessive!

File: src/components/nav/Nav.css (hunk 2 above)
Comment (AsherSpurr) [reply src/components/nav/Nav.css:20]: Oh yes 100% plan on doing this!

File: src/components/home/Home.tsx (hunk 1 above)
Comment (AsherSpurr) [reply src/components/home/Home.tsx]: On it!

Comment (AsherSpurr) [reply src/components/home/Home.tsx]: Good to go

**Review by tednaphil — APPROVED** [review]

**Pull request merged**